packages/react/src/FormControl/FormControl.tsx

```diff
@@ -200,9 +201,11 @@ const FormControl = React.forwardRef<HTMLDivElement, FormControlProps>(
                 React.isValidElement(child) &&
                 !expectedInputComponents.some(inputComponent => child.type === inputComponent),
             )}
-            {slots.validation ? (
-              <ValidationAnimationContainer show>{slots.validation}</ValidationAnimationContainer>
-            ) : null}
+            <Status>
```

Comment:
I'm curious if we'd want to provide an option to turn the live region off. For example, if multiple form fields produce an error at the same time they will all be announced through the live region, which isn't ideal and is the default behavior outside of this PR.

I'm not too sure on how we should give the option to turn live regions off. Should it be a prop, or should we handle this ourselves internally?

Cc: @joshblack, would love your input here 👀

Reply:
Ooof, that's a great question 😅 This is likely something that needs to be handled at the `<form>` level in order to coordinate. What specific scenarios are we expecting validation feedback to be announced?

Reply:
@TylerJDev so it seems like this case is for when feedback for an input is "instant" versus when it is submitted as part of a form, is that right? 👀

Reply:
Yup, exactly! I think the cases where all forms would suddenly show an error on submit is rare as that's the purpose of the inline validation, but there is still a chance that it could happen. Part of me feels like this might not be needed now, but something we might want to add later 🤔